packages/vue-components/src/Tab.vue

```diff
@@ -84,6 +83,14 @@ export default {
         margin: 0;
     }
 
+    .tab-pane.hide {
```

Comment:
would `tab-pane-hide` work instead? To avoid adding empty classes below to increase the specificity

Reply:
Seems like there is a css rule in bootstrap, `.tab-content > .active` that is more specific than `.tab-pane-hide`. We can also override that by using `.tab-content > .tab-pane-hide` in our scoped css but that would mean having to add `.tab-content > .printable-tab-pane` in the print rule as well. Should we go with this as this doesn't seem to resolve the empty classes issue?

Reply:
How about using `<style scoped>`? It adds `data-******` attributes under the hood and may be sufficient to override `.tab-content > .active`'s specificity.

Reply:
Thanks this works 👍 I've updated the PR accordingly.